src/Jobs/Recruit.Vacancies.Jobs/EditVacancyInfo/EditVacancyInfoUpdater.cs

```diff
@@ -1,22 +1,28 @@
 ﻿using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Microsoft.Extensions.Logging;
+using System.Linq;
 using System.Threading.Tasks;
 
 namespace Esfa.Recruit.Vacancies.Jobs.EditVacancyInfo
 {
     public class EditVacancyInfoUpdater
     {
         private readonly IJobsVacancyClient _client;
+        private readonly ILogger<EditVacancyInfoUpdater> _logger;
 
-        public EditVacancyInfoUpdater(IJobsVacancyClient client)
+        public EditVacancyInfoUpdater(IJobsVacancyClient client, ILogger<EditVacancyInfoUpdater> logger)
         {
             _client = client;
+            _logger = logger;
         }
 
         internal async Task UpdateEditVacancyInfo(string employerAccountId)
         {
-            var legalEntities = await _client.GetEmployerLegalEntitiesAsync(employerAccountId);
+            var legalEntities = (await _client.GetEmployerLegalEntitiesAsync(employerAccountId)).ToList();
```

Comment:
Don't think ToList is needed.

Reply:
It is enumerated multiple times isn't it?